core/src/SYCL/Kokkos_SYCL_Parallel_Range.hpp

```diff
@@ -94,9 +95,13 @@ class Kokkos::Impl::ParallelFor<FunctorType, Kokkos::RangePolicy<Traits...>,
       FunctorWrapperRangePolicyParallelFor<Functor, Policy> f{policy.begin(),
                                                               functor};
       sycl::range<1> range(policy.end() - policy.begin());
+      cgh.depends_on(memcpy_events);
```

Comment:
Do we still need a `std::vector` of events here?

Reply:
No, not really. I just changed it for consistency with the other versions of sycl_direct_launch. I can revert if you prefer.

Reply:
No problem I'll change this & other occurrences.